widget/embedded/src/components/ConfirmWalletsModal/ConfirmWallets.styles.ts

```diff
@@ -118,7 +118,14 @@ export const ConfirmButton = styled('div', {
 
 export const StyledTextField = styled(TextField, {
   backgroundColor: '$neutral100',
-  padding: '$0 $15 $15 $15',
+  paddingLeft: '$15',
+  variants: {
+    show: {
+      false: {
+        visibility: 'hidden',
+      },
+    },
+  },
```

Comment:
The padding of the text field doesn't align with the UI design. Additionally, there's a shift in the position of the paste icon when opening and closing the collapsible.

Reply:
Fixed the layout shift. please check if you are happy with the paddings.